Signal/src/ViewControllers/CallViewController.swift

```diff
@@ -338,16 +338,30 @@ class CallViewController: OWSViewController, CallObserver, CallServiceObserver {
 //                                                action:#selector(didPressTextMessage))
         audioSourceButton = createButton(imageName:"audio-call-speaker-inactive",
                                           action:#selector(didPressAudioSource))
+        audioSourceButton.accessibilityLabel = NSLocalizedString("AUDIO_SOURCE_LABEL",
```

Comment:
Can you prefix all these string names with `CALL_VIEW_` e.g. `CALL_VIEW_AUDIO_SOURCE_LABEL` this can help make sure strings in a large codebase don't collide, and also this groups all the call view strings together for our translators.

Reply:
Fixed 👍